apps/credential-showcase-api-server/src/database/schema/stepAction.ts

```diff
@@ -0,0 +1,19 @@
+import { pgTable, uuid, varchar } from 'drizzle-orm/pg-core';
+import { steps } from './step';
+import { relations } from 'drizzle-orm';
+
+// TODO this needs to become a base entity
+export const stepActions = pgTable('stepAction', {
+    id: uuid('id').notNull().primaryKey().defaultRandom(),
+    type: varchar({ length: 255 }).notNull(),
+    title: varchar({ length: 255 }).notNull(),
+    text: varchar({ length: 255 }).notNull(),
+    stepId: uuid('step_id').references(() => steps.id,{ onDelete: 'cascade' }).notNull()
+});
```

Comment:
Is there support for AriesOOBAction?  The openapi file has
```
      discriminator:
        propertyName: actionType
        mapping:
          ARIES_OOB: '#/components/schemas/AriesOOBAction'
```

Reply:
this is another ticket